File: Modul-3/Soal-2/PRAK302-2410817110016-MuhammadAzriRahman.py
bil = -1

while (bil > 100 or bil < 0): 
    bil = int(input("Masukan nilai 0-100:"))
    
    if (bil >= 80):
        print("A")
    elif (bil < 80 and bil >= 70):
        print("B")
    elif (bil < 70 and bil >= 60): 
        print("C")
    elif (bil < 60 and bil >= 50): 
        print("D")
    else:
        print("E")

File: Modul-3/Soal-2/PRAK302-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>

void main()
{
    int bil = -1;
    
    while (bil > 100 || bil < 0) {
        printf("Masukan nilai 0-100:");
        scanf("%d", &bil);
    }
    
    if (bil >= 80) {
        printf("A\n");
    }
    else if (bil < 80 && bil >= 70) {
        printf("B\n");
    }
    else if (bil < 70 && bil >= 60) {
        printf("C\n");
    }
    else if (bil < 60 && bil >= 50) {
        printf("D\n");
    }
    else {
        printf("E\n");
    }
}